### ISSUE TITLE
How to save generated plots on disk ? 

### ISSUE THREAD

**Issue by VinayChaudhari1996** [shap/shap#1651]
# Question 

Hello guys , 

I'm trying to **save SHAP result plot with .png or .jpg or .svg format.** But I didn't found method for same.
How canna I achieve that ? 

Thanks,

**Comment by YE-Qingtian** [comment]
Looking for the solution as well. 

in the original code, the function of shap.plot.force or shap.plot.beeswarm didn't return the plt object. 
Is there any way to savefig() without changing the original code?

**Comment by Momentum-TN** [comment]
@VinayChaudhari1996 @Viskym I don't know if you still looking for a solution. you can use the following method:
with setting `show` parameter to `False`, you can do all your matplotlib operations before `plt.show()`

from the [docs](https://shap.readthedocs.io/en/latest/generated/shap.plots.image.html#shap.plots.image) :

> show: bool
>
>   Whether matplotlib.pyplot.show() is called before returning. Setting this to False allows the plot to be customized further after it has been created.

example: 
```py
shap.image_plot(shap_numpy, -test_numpy, show=False)

# do all your matplotlib operations here before saving

# save your plot
plt.savefig('score.png', dpi=1080)
# display your plot
plt.show()
```

You can also use it for `shap.plot.force`.
I hope this is useful for you

**Comment by mphla** [comment]
@Momentum-TN This is correct, I tried it for a long time before, "show = False" is the key. My god ,thank you.
